manage.py

```diff
@@ -57,6 +58,17 @@ def import_data():
 
     logger.info('Starting to import OSM data from\n\t{}'.format("\n\t".join([str(p.resolve()) for p in osm_files])))
     parser.run_import(osm_files)
+
+    prewarm = app.config['POSTGRES_PREWARM']
+    if not is_testing() and prewarm == True:
+        tables = ['ops_pois', 'ops_categories', 'ops_tags']
```

Comment:
I tried adding the table names to the app config itself but because flasks app context is missing in `models.py` they can't be used there so I hardcoded them..

Reply:
Yep, thanks, done